src/store/FilingHistoryList/getters.ts

```diff
@@ -77,5 +77,23 @@ export default {
   /** Whether one document is downloading. */
   isLoadingOne (state: FilingHistoryListStateIF): boolean {
     return state.loadingOne
+  },
+
+  isAuthorizedToContinueOut (state: FilingHistoryListStateIF, rootGetters: any): boolean {
+    const ccoFilings = state.filings.filter(val => {
+      const exp = val.data?.consentContinuationOut?.expiry
+      if (exp) {
+        return true
+      }
+      return false
+    })
+    if (ccoFilings && ccoFilings.length > 0) {
+      const exp = ccoFilings[0].data?.consentContinuationOut?.expiry
+      const ccoExpiryDate = DateUtilities.dateToYyyyMmDd(new Date(exp))
```

Comment:
Please use the correct date utility to convert expiry into a Date object. That will ensure proper conversion and you won't need to create an object for it on line 94.

Out of curiousity, what is the expiry date format coming from Legal API?

Reply:
The expiry date format is something like "2023-11-09T08:00:00+00:00".

Reply:
OK, that's what we call "API format". See [apiToDate()](https://github.com/bcgov/business-filings-ui/blob/532caa1dc0ef4458abdfc723198642d8e6d3c7c8/src/services/date-utilities.ts#L197).

Also, can you confirm (with Argus or Vysakh or Kevin) that the date refers to a DAY (ie, the time is always 0700 or 0800)?

Reply:
(Background: sometimes a date is a DAY, and sometimes it's a date-time. In this case it looks like a day-only because the offset is exactly the Pacific-to-Greenwich offset. I prefer that day-only dates not have the time component, eg,  simply "2023-11-09", but the UI can handle it either way if it know what to expect.)

Reply:
it's always 0800

Reply:
Thanks.
It had better be 0700 for half the year 😉

Reply:
`ccoExpiryDate` and `currentDate` are now day-only in this getter. For example, "2023-11-09". Do you think I had better switch to another date utility?

Reply:
I think your use of `DateUtilities.daysFromToday()` is fine, but it needs 2 JS Date objects. I think you only need to convert `ccoExpiryDate` efficiently.

Reply:
I was thinking to use `DateUtilities.isDateFuture()`. In this case, only `ccoExpiryDate` is converted. But if the date is today, it will return false (expected true).

Reply:
You have a few options.

Once expiry is converted to a Date, you could also just compare against "now" right here.

Reply:
Yeah, I think that's what `DateUtilities.isDateFuture()` does, except it's not for day-only dates. So I choose to use `DateUtilities.daysFromToday()` for day-only dates, though it seems not very efficient.

Reply:
Sure.

And you can simplify the conversion of expiry (API date) to a JS Date object.

Reply:
I've modified the file and updated the unit test. 😃 Is it better?